### PR TITLE
[ADD] new module 'pos_session_summary';

### PR DESCRIPTION
**Pull request by legalsylvain**
Hi, here is a trivial module to display : 
- the total amount of transactions of a session ;
- the orders qty of a session ;

pos_session_list

thanks for your review, kind regards.

### CODE REVIEW

File: pos_session_summary/README.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Point of Sale - Total of transactions and Orders Quantity
```
Comment (pedrobaeza) [inline pos_session_summary/README.rst:6]: Put upper line

File: pos_session_summary/README.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```
Comment (pedrobaeza) [inline pos_session_summary/README.rst]: Put link according latest README: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: pos_session_summary/README.rst [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+Configuration
+=============
+
+No configuration is needed.
+
+Usage
+=====
+
+For further information, please visit:
```
Comment (pedrobaeza) [inline pos_session_summary/README.rst]: Add Try me on runbot button

File: pos_session_summary/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```
Comment (pedrobaeza) [inline pos_session_summary/__openerp__.py]: Put short header like https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

File: pos_session_summary/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    'summary': 'Point of Sale - Total of transactions and Orders Quantity',
+    'author': 'La Louve, Odoo Community Association (OCA)',
+    'website': 'http://www.lalouve.net/',
+    'depends': [
+        'point_of_sale',
+    ],
+    'data': [
+        'views/pos_session_view.xml',
+    ],
+    'installable': True,
```
Comment (pedrobaeza) [inline pos_session_summary/__openerp__.py]: Include license key

**Comment by legalsylvain** [comment]
Thanks @pedrobaeza for your review !
your comments have been taken into account.

regards.

File: pos_session_summary/README.rst [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+=============
+
+No configuration is needed.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to ...
```
Comment (pedrobaeza) [inline pos_session_summary/README.rst]: Explain this

**Comment by legalsylvain** [comment]
@pedrobaeza : updated again.

@all : Anybody has an idea about why Travis fails ? 
https://travis-ci.org/OCA/pos/jobs/106048056
The fail is on 'sale' module tests, 
openerp_test openerp.addons.sale.tests.test_sale_order: ` IOError: [Errno 2] command not found: 'wkhtmltopdf'

**Comment by pedrobaeza** [comment]
For v9, you need to add wkhtmltopdf compatibility in Travis, because now tests include report generation. Check https://github.com/odoomrp/odoomrp-wip/blob/9.0/.travis.yml#L12, https://github.com/odoomrp/odoomrp-wip/blob/9.0/.travis.yml#L18 and https://github.com/odoomrp/odoomrp-wip/blob/9.0/.travis.yml#L34-L36 for the needed changes.

**Comment by legalsylvain** [comment]
Ow thanks !

**Comment by legalsylvain** [comment]
@pedrobaeza : all is now green and fixed.

Thanks for your review.

Regards.

File: pos_session_summary/models/pos_session.py [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+
+class PosSession(models.Model):
+    _inherit = 'pos.session'
+
+    @api.multi
+    @api.depends('statement_ids.balance_end')
+    def _compute_total_amount(self):
+        for session in self:
+            total_amount = 0
```
Comment (pedrobaeza) [inline pos_session_summary/models/pos_session.py]: You can reduce this making:

```
session.total_amount = sum(session.mapped('statement_ids.balance_end'))
```

**Comment by legalsylvain** [comment]
Ping @POS contributors.

(@jcarlosmontoya, @alexis-via, @antespi, @sebastienbeau.)

**Review by pedrobaeza — APPROVED** [review]

**Comment by jweste** [comment]
+1

**Comment by legalsylvain** [comment]
Thanks for your review.

**Pull request merged**